connect/eaas/core/validation/validators/eventsapp.py

```diff
@@ -115,12 +115,24 @@ def _validate_events(context):
                 ),
             )
             continue
+        if len(event['statuses']) == 0:
```

Comment:
There are some events that doesn't require statuses like product actions.
You have to add to the condition:
```
if definitions[event['event_type']]['object_statuses'] and not event['statuses']
```